demos/3d_spherical/3d_spherical.py

```diff
@@ -25,7 +25,7 @@
 # using the optional keyword argument *degree$=2$*.
 
 # +
-rmin, rmax, ref_level, nlayers = 1.22, 2.22, 4, 8
+rmin, rmax, ref_level, nlayers = 1.208, 2.208, 4, 8
```

Comment:
This is why this test is failing, probably? Probably need to update answers, if you've verified the new values are correct.

Reply:
Thanks @angus-g for sorting this out. Sorry for being so slow with this.